src/Compilers/CSharp/Portable/Symbols/Synthesized/GeneratedNameKind.cs

```diff
@@ -35,6 +35,8 @@ internal enum GeneratedNameKind
         AsyncBuilderField = 't',
         DynamicCallSiteContainerType = 'o',
         DynamicCallSiteField = 'p',
+        MethodGroupConversionCacheFrame = 'q',
+        MethodGroupConversionCacheDelegateField = 'r',
```

Comment:
@tmat Do you remember where the master list is?  A lot of letters are already taken.